src/internal/clusterstate/v2.1.0.go

```diff
@@ -16,7 +16,7 @@ var state_2_1_0 migrations.State = state_2_0_0.
 		return enterpriseserver.DeleteEnterpriseConfigFromEtcd(ctx, env.EtcdClient)
 	}).
 	Apply("create pfs cache v1", func(ctx context.Context, env migrations.Env) error {
-		return fileset.CreatePostgresCacheV1(ctx, env.Tx)
+		return storagedb.SchemaFilesetCacheV1(ctx, env.Tx)
```

Comment:
I think all of these functions should include `Create` in the name since they are creating database objects. Also, they are all creating tables rather than schemas, so we should change that too.